spec/posthog/feature_flag_spec.rb

```diff
@@ -0,0 +1,2951 @@
+require 'spec_helper'
+
+
+class PostHog
+
+  RSpec::Support::ObjectFormatter.default_instance.max_formatted_output_length = nil
+
+  describe 'local evaluation' do
+  
+    it 'evaluates person properties' do
+      api_feature_flag_res = {
```

Comment:
Note: rspec is _the_ most powerful testing library our there and we're using it here like it was jest - this is a missed opportunity for learning.

Want to do a call tomorrow (unrelated to this call) and pair on how to refactor all of this?